back-end/app.js

```diff
@@ -4,6 +4,16 @@ const app = express()
 require('dotenv').config()
 app.use(express.json())
 
+app.get('/tv_users', (req, res, next) => {
+    axios.get(`https://my.api.mockaroo.com/tv_users.json?key=`)
```

Comment:
Looks good Mason! The only thing is could you add in this after `key=`? `${process.env.MOCKAROO_KEY}` (so the axios URL looks like the others in `app.js`